src/components/Properties.astro

```diff
@@ -1,8 +1,49 @@
-<div class="my-6">
+---
+const { heading = 'Attributes' } = Astro.props;
+---
+
+<div data-attributes class="my-6">
+  <div class="flex justify-between items-center not-prose py-2">
+    <h3 class="scroll-mt-10 type-subtitle-1">{ heading }</h3>
```

Comment:
Suggestion: Are these ever clickable titles? If not we probably don't need the scroll margin.

Reply:
I thought a followup change might be to add ids to these fields so we can index them and include a search entry for them. Maybe its not a requirement though so will remove for now.